BeerOverflowWindowsApp/MainWindow.cs

```diff
@@ -21,13 +21,17 @@ public partial class MainWindow : Form
         private readonly string _defaultLatitude = ConfigurationManager.AppSettings["defaultLatitude"];
         private readonly string _defaultLongitude = ConfigurationManager.AppSettings["defaultLongitude"];
         private readonly string _defaultRadius = ConfigurationManager.AppSettings["defaultRadius"];
+        MapWindow mapForm;
 
         public MainWindow()
         {
             InitializeComponent();
             _barRating = new BarRating();
             LatitudeTextBox.Text = _defaultLatitude;
             LongitudeTextBox.Text = _defaultLongitude;
+            var latitude = Convert.ToDouble(_defaultLatitude, CultureInfo.InvariantCulture);
+            var longitude = Convert.ToDouble(_defaultLongitude, CultureInfo.InvariantCulture);
```

Comment:
Instead of this, maybe the map window could check if current location is null and then get the default location from config files? It would be like a backup location if something fails.

Reply:
Will do